db/helpers.js

```diff
@@ -339,7 +354,7 @@ module.exports.getInvitees = function(unhide=false) {
       var promiseArray = users.map(eachUser => eachUser.fetch({withRelated: [
         {invitedToBoards: function(query) {
           //query.whereRaw('last_email = "null"');
-          query.where('last_email', null).orWhereRaw(`last_email < now()-INTERVAL '2 days'`);
+          query.where('last_email', null).orWhereRaw('last_email < now()-INTERVAL \'2 days\'');
```

Comment:
i think the backtick ` is needed. or else it will interfere with the inner single quotes. and raw query seems to only like inner single quotes

Reply:
it certainly is!